db/schema.rb

```diff
@@ -55,6 +55,7 @@
     t.string "name"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "status", default: 0
```

Comment:
Should have added this in the beginning haha, my bad. Probably would have helped everyone